src/cli/mod.rs

```diff
@@ -340,6 +340,17 @@ enum Command {
         consolidate_deprecated_keychains: bool,
         #[clap(short = 'i', long = "interval-trigger")]
         interval_trigger: Option<u32>,
+        #[clap(short = 'm', long = "manual", default_value = "false")]
```

Comment:
I'm confused about there being both a default and `Option` type. Shouldn't It be just one or the other?